src/components/view/Details.js

```diff
@@ -52,10 +99,8 @@ var Details = React.createClass({
     var self = this;
     var task = self.props.task;
     var renderedSubtasks = self.state.subTasks && self.renderSubtasks();
-    // get remaining time
-    // separate main task from steps
-    // add links
-    // add editing
+    var classList = 'pictogram-icon wundercon gray mr1';
+    classList += (task.completed ? ' icon-checkbox-filled': ' icon-checkbox');
```

Comment:
use the classnames module, it will make conditional styles more sane to deal with